packages/open-next/src/adapters/image-optimization-adapter.ts

```diff
@@ -1,26 +1,30 @@
-import path from "node:path";
+import { IncomingMessage, ServerResponse } from "node:http";
 import https from "node:https";
+import path from "node:path";
 import { Writable } from "node:stream";
-import { IncomingMessage, ServerResponse } from "node:http";
+
 import { GetObjectCommand, S3Client } from "@aws-sdk/client-s3";
 import type {
   APIGatewayProxyEvent,
-  APIGatewayProxyEventV2,
-  APIGatewayProxyResultV2,
   APIGatewayProxyEventHeaders,
   APIGatewayProxyEventQueryStringParameters,
+  APIGatewayProxyEventV2,
+  APIGatewayProxyResultV2,
 } from "aws-lambda";
 // @ts-ignore
 import { defaultConfig } from "next/dist/server/config-shared";
-// @ts-ignore
-import type { NextUrlWithParsedQuery } from "next/dist/server/request-meta";
 import {
   imageOptimizer,
   ImageOptimizerCache,
   // @ts-ignore
 } from "next/dist/server/image-optimizer";
+// @ts-ignore
+import type { NextUrlWithParsedQuery } from "next/dist/server/request-meta";
+
+import { RequestHandler } from "../types.js";
+import { awsLogger, debug, error } from "./logger.js";
+import { streamError, streamSuccess } from "./streaming.js";
```

Comment:
@sladg I'm getting type errors when I run `npm run build` in the `packages/open-next` folder. I don't see the `types.ts` or `streaming.ts` files in this PR. Did you mean to include your streaming changes in this PR? If so, you might have missed the files.

Reply:
Thanks for heads up! Seems like bad rebase from `main` branch.